### ISSUE TITLE
Having the component twice on a page

### ISSUE THREAD

**Issue by cdietrich** [TypeFox/monaco-components#64]
Hi, do you have a hint how to have the component twice on a page.
am currently getting

```
Uncaught (in promise) Error: Services are already initialized
    at checkServicesNotInitialized (lifecycle.js:68:15)
    at initialize2 (services.js:84:5)
    at importAllServices (monaco-vscode-api-services.ts:101:11)
    at initServices (monaco-vscode-api-services.ts:29:15)
    at LanguageClientWrapper.init (languageClientWrapper.ts:93:15)
    at MonacoEditorLanguageClientWrapper.init (wrapper.ts:62:42)
```

**Comment by kaisalmen** [comment]
Hi @cdietrich did you wait everything properly. There is a safe-guard for this during init. It works, see:
https://github.com/TypeFox/monaco-components/blob/main/packages/examples/src/langium/wrapperStatemachine.ts

**Comment by kaisalmen** [comment]
... or is this working for the wrapper and not for the react component? Then this is a new issue.

**Comment by cdietrich** [comment]
We are using the react component yes

**Comment by montymxb** [comment]
@cdietrich are you using [strict mode](https://react.dev/reference/react/StrictMode)? That will trigger a 2x render, and it will make impurity issues pretty evident.

**Comment by cdietrich** [comment]
am not in strict mode. with having the component only once i dont see the problem

**Comment by kaisalmen** [comment]
@cdietrich seems to be a bug we need to fix. Thanks for reporting.

**Comment by kaisalmen** [comment]
Fixed will be contained in next release (see https://github.com/TypeFox/monaco-languageclient/pull/600)
https://github.com/TypeFox/monaco-languageclient/pull/600/commits/9f49bea75114cf0785f12cb0d3bbde03d08604af

Update preview releases will come in a few minutes...

**Comment by cdietrich** [comment]
@kaisalmen in my example this still does not seem to work.
https://github.com/cdietrich/my-monaco-editor-react-example/blob/660e1b2cd3f189bd26e9ef1804e54090f4b49e83/src/main.tsx#L7

replace this with <div><App/><App/></div> does show the same error

**Comment by cdietrich** [comment]
@kaisalmen maybe this one slipped through too

**Comment by cdietrich** [comment]
looks better now, but somehow the two editors seem to talk to the same language client

**Comment by kaisalmen** [comment]
> the two editors seem to talk to the same language client. but maybe also problem on my side.

There is #16 that is still unresolved. If you have two editor configured with the same languageId you run into this problem.

**Comment by cdietrich** [comment]
thx for the heads up. a quick change of the 2nd language id does not seem to help

**Comment by kaisalmen** [comment]
@cdietrich ok, we need to implement #16 I will now move all open issues to the other repo.
